src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts

```diff
@@ -23,7 +23,17 @@ const MapEventToProcess = {
     [MLGYM_EVENT.EVALUATION_RESULT]: (dataToRedux: DataToRedux, data: any) => {
         const evalResPayload = data as unknown as EvaluationResultPayload;
         dataToRedux.evaluationResultsData = handleEvaluationResultData(initialStateForGraphs, evalResPayload);
-        dataToRedux.latest_split_metric = evalResPayload;
+        // TODO: if handleEvaluationResultData is gonna get improved, then move the following inside of it
```

Comment:
this seems like a TODO for #145?

Reply:
If not --> such todos should be become "todo" issues in Github ;-)

Reply:
issue #152 and it has a draft PR #154